docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -156,6 +157,13 @@ priority
   *(optional, array)* Specifies a set of rate limit configurations that could be applied to the
   route.
 
+.. _config_http_conn_man_route_table_route_include_vh:
+
+include_vh_rate_limits
+  *(optional, boolean)* Specifies if the rate limit filter should include the virtual host rate limits.
+  The default value is false if
```

Comment:
I might be just a bit more clear here that the default is true if there is no route local rate limit. I could see this being somewhat confusing to new users.